lev-data/fix_lev_death_registration_ids.sql

```diff
@@ -0,0 +1,6 @@
+CREATE SEQUENCE death_registration_ids;
+
+UPDATE death_registration_v1
+SET id=nextval('death_registration_ids');
+UPDATE lev.public.death_registration_v1
+SET data = jsonb_set(data::jsonb, '{id}', to_jsonb(id));
```

Comment:
I think I'd expect something like 
```sql
ALTER TABLE death_registration_v1 ALTER COLUMN id SET DEFAULT nextval('death_registration_ids');
```
here?

Reply:
Have added it in - although in the lambda we won't be using this default so the id's match up